apps/src/templates/teacherHomepage/ManageSectionsCollapsible.story.jsx

```diff
@@ -39,6 +39,7 @@ export default storybook => {
                 sectionCode: "HPRWHG"
               },
             ]}
+            codeOrgUrlPrefix = "http://localhost:3000/"
```

Comment:
Storybook, in my experience so far,  isn't a good place to test links because you usually end up with a ` 'X-Frame-Options' to 'sameorigin'.` error. I assigned `codeOrgUrlPrefix = "http://localhost:3000/"`  in the stories because then it's clear to whomever uses this component later what is expected to be passed for that value. Make sense?